docs/docs/code/python/README.md

```diff
@@ -345,14 +341,13 @@ You have full access to read and write both files in `/tmp`.
 ```python
 import requests
 
-def handler(pd: "pipedream"):
-  # Download the Python logo
-  r = requests.get('https://www.python.org/static/img/python-logo@2x.png')
+# Download the Python logo
+r = requests.get('https://www.python.org/static/img/python-logo@2x.png')
 
-  # Create a new file python-logo.png in the /tmp/data directory
-  with open('/tmp/python-logo.png', 'wb') as f:
-    # Save the content of the HTTP response into the file
-    f.write(r.content)
+# Create a new file python-logo.png in the /tmp/data directory
+with open('/tmp/python-logo.png', 'wb') as f:
+  # Save the content of the HTTP response into the file
```

Comment:
Can you update to 4 spaces for indentation (required by [PEP-8](https://peps.python.org/pep-0008/)). Is there an extension that will apply those rules for us?